File: plugins/OrchidKSS/src/main/java/com/eden/orchid/kss/KssResourceSource.java
package com.eden.orchid.kss;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;

public class KssResourceSource extends PluginResourceSource {

    @Inject
    public KssResourceSource(OrchidContext context) {
        super(context, 20);
    }
}

File: plugins/OrchidKSS/src/main/kotlin/com/eden/orchid/kss/KssResourceSource.kt
package com.eden.orchid.kss

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject

class KssResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 20)

